Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/ButtonHoloLens1.prefab

```diff
@@ -388,9 +390,10 @@ MonoBehaviour:
   InputActionId: 0
   IsGlobal: 0
   Dimensions: 1
+  StartDimensionIndex: 0
   CanSelect: 1
   CanDeselect: 1
-  VoiceCommand: Button
```

Comment:
Do we want this to keep its voice command?

Reply:
Re-assigned the voice command.